Core/GDCore/Project/Project.cpp

```diff
@@ -615,6 +615,31 @@ void Project::UnserializeFrom(const SerializerElement& element) {
       propElement.GetStringAttribute("macExecutableFilename");
   useExternalSourceFiles =
       propElement.GetBoolAttribute("useExternalSourceFiles");
+  
+  /*
+  SerializerElement& extensionPropertiesElement = propElement.GetChild("extensionProperties");
+  for(std::pair<gd::String, std::shared_ptr<gd::SerializerElement>> extension : extensionPropertiesElement.GetAllChildren()) {
+    std::shared_ptr<PlatformExtension> platformExtension = GetCurrentPlatform().GetExtension(extension.first);
+    for(std::pair<gd::String, std::shared_ptr<gd::SerializerElement>> property : extension.second->GetAllChildren()) {
+      gd::PropertyDescriptor propertyInstance = *platformExtension->GetProperty(property.first).Clone(); // Take the extension's property as "template" to keep the metadata.
+      propertyInstance.UnserializeValuesFrom(*property.second); // Apply the saved value
+      extensionProperties[extension.first][property.first] = propertyInstance; // Put it in the project.
+    }
+  }
+  */
+  
+  SerializerElement& allExtensionsPropertiesElement = propElement.GetChild("extensionProperties");
+  for(std::shared_ptr<PlatformExtension> extension : GetCurrentPlatform().GetAllPlatformExtensions()) { // We do not iterate over serialized object to include also new properties and remove old removed ones.
+    gd::String extensionName = extension->GetName(); // Cache the name to not call the function again...
+    SerializerElement& extensionElement = allExtensionsPropertiesElement.GetChild(extensionName); // ... and the serializer element too
+    for(std::pair<const gd::String, gd::PropertyDescriptor>& property : extension->GetAllProperties()) {
+      gd::PropertyDescriptor propertyInstance = *property.second.Clone(); // Take the extension's property as "template" to keep the metadata.

```

Comment:
You won't need to do all that complicated stuff. The project should just store a bunch of key-values. Forget about properties in the project.

Reply:
This might still be useful in the method that returns the properties given an extension (which can be used in `propertiesMapToSchema` as the second argument)